docs/proposals/2019-09-30 - Content by reference in Link Lists.md

```diff
@@ -0,0 +1,64 @@
+## Problem
+Currently, editors can only add URLs in `interstitial_link` to Link Lists. This presents a few issues:
+1. it's not possible to use a Link List until all of the desired content is published
+2. it's not as straightforward to fetch additional information about the story with just the URL
+
+## Proposal
+We propose to add additional traits, `Story`, `Video`, and `Gallery`, to `interstitial_link`, so we can add more types of elements by reference to link lists in addition of `image`.
```

Comment:
>How does this solve the actual problem? 
According to the Problem, this will enable user to add either of story, video, or gallery to Link List while either of story, video, or gallery is still working in progress/unpublished by using reference instead of the URLs, which is generated after it is published. 
>What happens if the Story or Video or Gallery isn't published?
User should still be able to add them into Link List.